src/persist-client/src/batch.rs

```diff
@@ -404,7 +404,7 @@ pub(crate) const STRUCTURED_ORDER_UNTIL_SHARD: Config<&'static str> = Config::ne
 
 pub(crate) const STRUCTURED_KEY_LOWER_LEN: Config<usize> = Config::new(
     "persist_batch_structured_key_lower_len",
-    0,
+    256,
```

Comment:
Ben knows best, but I would say this should probably stay at 0 since we're not out to 256 yet in all of prod

Reply:
Agreed!